scripts/ubuntu-install-test-no-nix-docker

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# This script is to test the ubuntu installation instructions locally on
+# non-ubuntu machines by running through the installation and tests inside an
+# ubuntu docker contrainer.
+set -e
+
+# Create a temporary copy of the repository to be mounted into the container.
+REPO_COPY_DIR="$(mktemp -d)"
+
+# Remove repo copy on exit, error etc.
+trap "exit" INT TERM
+trap cleanup EXIT
+cleanup(){
+    echo "Cleaning up repo copy: $REPO_COPY_DIR, need sudo"
+    sudo rm -rf "$REPO_COPY_DIR"
+}
+
+git clone --recursive . "$REPO_COPY_DIR"
+
+# Run the installation and tests inside the container.
```

Comment:
Since we are using docker anyway, instead of mounting a dir, couldn't we copy the build script into an ubuntu container and do all the git clone inside the container? This way everything is ephemeral and no need for external temp dirs.

Reply:
It's done like this now so you can get local changes that aren't yet on github inside the container. The script is just for local debugging.